card/src/main/java/com/adyen/checkout/card/CardConfiguration.kt

```diff
@@ -160,37 +160,6 @@ class CardConfiguration private constructor(
             return this
         }
 
-        /**
-         * Set if the CVC field should be hidden from the Component and not requested to the shopper on a regular
-         * payment.
-         * Note that this might have implications for the risk of the transaction. Talk to Adyen Support before enabling
-         * this.
-         *
-         * Default is false.
-         *
-         * @param hideCvc If CVC should be hidden or not.
-         * @return [CardConfiguration.Builder]
-         */
-        fun setHideCvc(hideCvc: Boolean): Builder {
```

Comment:
These are breaking changes

Reply:
Reverted back to the original configurations